packages/cerebral-forms/package.json

```diff
@@ -26,7 +26,9 @@
     "cerebral": "^2.0.0-0"
   },
   "devDependencies": {
-    "cerebral": "^2.0.0-0"
+    "cerebral": "^2.0.0-0",
+    "babel-preset-es2015": "^6.16.0",
```

Comment:
i had to ad presets like this to make it work with windows (well at least the http-provider- package which i tested)
not sure how this presets gets picked up (i believe as soon as there is a config in .babelrc, right?)